nautobot/docs/user-guide/platform-functionality/jobs/index.md

```diff
@@ -87,24 +41,6 @@ An administrator or user with `extras.delete_job` permissions *may* delete such
 
     ![Adding the run action to a permission](../../../media/admin_ui_run_permission.png)
```

Comment:
Minor nit here, would be great to let users know where that screenshot is taken from, breadcrumbs style.